include/variables.h

```diff
@@ -500,52 +195,14 @@ extern s16 gLensFlareScale;
 extern f32 gLensFlareColorIntensity;
 extern s16 gLensFlareScreenFillAlpha;
 extern LightningStrike gLightningStrike;
-//extern ? D_8015FDF8;
 extern MapData* gMapData;
-//extern ? D_80160020;
-//extern ? D_801600C0;
-extern u8 D_80161010;
-extern u8 D_801610D4;
-extern u8 D_801610D5;
-extern u8 D_801610DA;
-//extern ? D_80161118;
-//extern ? D_8016111A;
-//extern ? D_8016111C;
-//extern ? D_80161120;
-//extern ? D_8016112C;
-//extern ? D_80161138;
-//extern ? D_8016113C;
-//extern ? D_80161160;
-//extern ? D_8016116E;
-//extern ? D_80161170;
-//extern ? D_8016117E;
-//extern ? D_80161180;
-//extern ? D_80161240;
-//extern ? D_801612D0;
-//extern ? D_801612D2;
-//extern ? D_801612EC;
 extern f32 gBossMarkScale;
 extern PauseMapMarksData* gLoadedPauseMarkDataTable;
 extern s32 gTrnsnUnkState;
 extern Color_RGBA8_u32 D_801614B0;
-extern SpeedMeter gSpeedMeter;
```

Comment:
should be renamed to sSpeedMeter in the relevant file

Reply:
this symbol doesn't actually exist, probably meant to be `D_801664D0` (?) but I'll leave that to someone else